javascript/node/selenium-webdriver/firefox.js

```diff
@@ -592,7 +592,7 @@ class Driver extends webdriver.WebDriver {
    * implementation.
    * @override
    */
-  setFileDetector() { }
```

Comment:
Can we have the linting changes in a separate commit within this PR so that it is easier to spot the functional changes